test/k8s/simple_test.go

```diff
@@ -39,40 +43,48 @@ data:
 				Setup(cluster)
 			Expect(err).ToNot(HaveOccurred())
 			opts = append(opts,
-				WithHelmOpt("controlPlane.secrets[0].Env", "KMESH_LICENSE_INLINE"),
-				WithHelmOpt("controlPlane.secrets[0].Secret", "kong-mesh-license"),
-				WithHelmOpt("controlPlane.secrets[0].Key", "license.json"),
-			)
+				WithCtlOpts(map[string]string{
+					"--license-path": license,
```

Comment:
I don't think we need to manually create secret and namespace in this case. I think kumactl do it whenever we pass license-path